hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/pipeline/StreamStageImpl.java

```diff
@@ -131,7 +131,7 @@ public StreamStageImpl(
 
     @Nonnull @Override
     public <R> StreamStage<R> rollingAggregate(@Nonnull AggregateOperation1<? super T, ?, ? extends R> aggrOp) {
-        return groupingKey(constantKey()).rollingAggregate(aggrOp, (k, v) -> v);
+        return groupingKey(constantKey(name().hashCode())).rollingAggregate(aggrOp, (k, v) -> v);
```

Comment:
is it a good idea to use `name`? it's always generated to the same value if not explicitly set (i.e. rolling-aggregate)

Reply:
Hmm, true, in this place it's not unique, it's made unique later when calling `createDag`.